### ISSUE TITLE
Is the training setup for the master network here

### ISSUE THREAD

**Issue by Dboingue** [glinscott/nnue-pytorch#141]
I don't know where to ask. this seems best related. So main question above.  motivation below.  source code last resort for documentation, but might be the less noisy too.  problem is where.  but if below questions can get answer before the main one, That would be great.

Also, any documentation about the training setup of master, if SL or RL, and what kind of policy used for self-learning, if that is what RL means for the master NNue (can it be also called NNue).  if no wiki on the master training repo.

I have a lot of non programming questions about what is actually done by the code at a mathematical or algorithmic level. 
If the master is using RL, then, leaving aside all the speed up goodies of NNue, and focusing on input-output, self-game batch exploration process, and policy involvement. are also my concern. I want to compare the master to lc0 not from an engine competition but from a machine learning framework point of view.  

if SL then the policy design question become database construction or selection and sub-sampling, and target definition for the global loss function.  there question are independent of how implemented the training is.  And that is why I wish to avoid source code as documentation. if possible.  

I am asking here, because for now, on github, in the SF repo.  the 2020, SF12 first code change introducing NNue, has existence of the master net as premise.  and focused on the transformers.  and their training with the master as "oracle". for positions database, huge. but not characterized there.  ( i also would like to know about those, are they related to the master training setup?). Also, there is talk there of high imbalance positions, so that i conclude the training is not pre-selecting material neutral position in the database setup.  

Can anyone course correct my inquiry line. here.  Thanks you much in advance.

Less important question, but asking, in case it is easy to clarify.
also, if NNue can also do high imbalance in material.  why are they only used for neutral one.  i thought it was an accuracy versus speed compromise, because of the cost of NNue evaluation per position compared to classic static eval.  But the final eval printout in debug mode, puzzles me.  as it does not seem to be either the NNue or the classical score printout. with tree search of not.

**Comment by vondele** [comment]
The master branch of the repo is essentially what is used for training the master net, using the command here https://github.com/glinscott/nnue-pytorch#current-recommended-training-invocation or the procedures mentioned in the Stockfish commits that introduce improved nets. For some of the question you ask it would be best to join the discord server and look for the NNUE channel.  https://discord.gg/aefaxmq

**Comment by Dboingue** [comment]
if this serves for others having same question maybe leave open. but otherwise my future inquiries have somewhere to go. thanks.
